static/js/TopicPage.jsx

```diff
@@ -585,9 +585,11 @@ const TopicPage = ({
 
     useEffect( ()=> {
     // hack to redirect to temporary sheet content on topics page for those topics that only have sheet content.
-        if (!Sefaria.is_moderator && !topicData.isLoading && Object.keys(topicData.tabs).length == 0){
-        const interfaceIsHe = Sefaria.interfaceLang === "hebrew"
-        const topicPath = interfaceIsHe ? topicTitle.he : topicTitle.en;
+        if (!Sefaria.is_moderator && !topicData.isLoading && Object.keys(topicData.tabs).length == 0 .length == 0 && topicData.subclass != "author"){
+        const interfaceIsHe = Sefaria.interfaceLang === "hebrew";
+        const interfaceLang = interfaceIsHe ? 'he' : 'en';
+        const coInterfaceLang = interfaceIsHe ? 'en' : 'he';
+        const topicPath = topicTitle[interfaceLang] || topicTitle[coInterfaceLang];
```

Comment:
Does this work? What if the topic has only an English title but interface is Hebrew? then the filter will be `stopics_he=<English title>`.